docs/src/documentation/06-kiwi-use/02-content/04-specific-areas/social-media/02-visual-treatment.mdx

```diff
@@ -1,3 +1,205 @@
 ---
-title: Empathetic
+title: Visual treatment
 ---
+
+import Grid from "@kiwicom/orbit-components/lib/utils/Grid";
+
+**_As the old Czech proverb goes: "Don't just say it, show it!"_**
+
+The right imagery makes our content much more powerful and engaging.
+It also helps us to differentiate from our competitors.
+
+Our photography should always have a sense of optimism.
+The subjects and environments should feel authentic.
+We shouldn't be afraid of showing **real faces and emotions.**
+People should be **unposed** and their **composition** within the environment **natural**.
+Lighting, color and tone should be **unfiltered and realistic \#NoFilter!**
+
+Each image should establish an emotional connection with our audience,
+showing them the world Kiwi.com can offer.
+
+## Our destinations...
+
+**ARE** real, authentic and full of wanderlust.
+Images don't just show the place but the _details_ of the people and the story.
+
+<Grid columns="200px 200px 296px;" rows="minmax(100px, auto) minmax(100px, auto);" gap="10px;">
+  <img
+    src="../../../../images/social-media/destination-island.png"
```

Comment:
I thought the images had been loading. I certainly saw them for a while. 😞 Probably when I changed from markdown to html tags.

Anyway, it all doesn't work anyway because to the `gatsby-remark-images` stuff and extra divs making the grid not align. I'll add a component and move the images back into markdown.

We can introduce link checkers that check whether the file exists. I think that's a more general solution. I'm not sure there will be lots of need for other <img> tags.

Reply:
> Anyway, it all doesn't work anyway because to the gatsby-remark-images stuff and extra divs making the grid not align. I'll add a component and move the images back into markdown.

We can always fix that with a custom component wrapping Orbit's `Grid`. I'm saying this before identifying what the problem is, but changing the authoring tactic because of a technical limitation is always a bad thing. To me this looks good, I'll let you know after I figure out what went wrong in the CSS department.

Reply:
The problem was the setting of the margins on children. But I think that the styling was removed as part of the `gatsby-remark-images` processing. So even if we wrapped the whole thing to solve the margins, it wouldn't solve the underlying issue. Having a custom component to wrap the images (really just divs there with an easier API) seems like enough to me. :shrug: